src/Tests/Blockcore.Features.Wallet.Tests/Blockcore.Features.Wallet.Tests.csproj

```diff
@@ -29,6 +29,7 @@
   <ItemGroup>
     <ProjectReference Include="..\..\Features\Blockcore.Features.Wallet\Blockcore.Features.Wallet.csproj" />
     <ProjectReference Include="..\..\Features\Blockcore.Features.RPC\Blockcore.Features.RPC.csproj" />
+    <ProjectReference Include="..\..\Networks\Blockcore.Networks.XRC\Blockcore.Networks.XRC.csproj" />
```

Comment:
Adding a specific network is the wrong approach, it should be generic, network agnostic